extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -201,7 +180,9 @@ Often per-texel feature IDs provide finer granularity than per-vertex feature ID
 }
 ```
 
-`texture` is a glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `channels` must be a single channel (`"r"`, `"g"`, `"b"`, or `"a"`). Furthermore, feature IDs must be whole numbers in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table. Texture filtering should be disabled when accessing feature IDs.
+The `featureId` entry for a feature ID texture extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. Each `channel` must be a positive integer corresponding to a channel of the source texture. Feature IDs must be whole numbers — stored in linear space — in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table.
```

Comment:
I think we should clarify that if the image is an RGBA image, channel 0 is R, channel 1 is green, and so on.

Reply:
Done ✅